### DISCUSSION TITLE
Use of Parallel Plugin in Astro

### DISCUSSION THREAD

**Discussion by krishnaTORQUE** [swup/swup#915] [Help]
### Describe your problem 🧐

How to use parallel plugin in astro?

swup config in astro
```
swup({
            theme: "fade",
            globalInstance: true,
            preload: true,
            smoothScrolling: true,
            animateHistoryBrowsing: true,
            containers: ["#swup"],
            plugins: [new SwupParallelPlugin()],
        }),
```

html
```
<section>
            <main id="swup" class="transition-slide">
                <slot />
            </main>
        </section>
```

page
```
<a href="/about" data-swup-animation="right">Next</a>

```

### What is your question? 🙋‍♀️

How to use parallel plugin in astro?

### URL to minimal reproduction 🔗

None

### Checked all these? 📚

- [X] I have read and searched the [official docs](https://swup.js.org)
- [X] I have checked [discussions](https://github.com/orgs/swup/discussions) and [existing issues](https://github.com/issues?q=is:issue+org:swup) for answers to my question
- [X] I agree to follow this project's [Code of Conduct](https://github.com/swup/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have provided all necessary information to the best of my knowledge

**Comment by daun** [comment]
If you look at the readme, you'll find a section on [parallel animations](https://github.com/swup/astro?tab=readme-ov-file#configparallel) that should point you in the right direction.

**Comment by krishnaTORQUE** [comment]
@daun thanks
I made changes according to readme but still nothing.
```
parallel: true,
containers: ["main"],
parallel: ["main"],
```

**Comment by krishnaTORQUE** [comment]
Even I checked this code but unable to find the issue
`https://replit.com/@philippdaun/Swup-Demo-Slideshow`

**Comment by krishnaTORQUE** [comment]
What I am trying to do is fade all of the page but slide in parallel when needed.

**Comment by hirasso** [comment]
@krishnaTORQUE you‘ll need to take the time and create a minimal reproduction of your issue.

**Reply by krishnaTORQUE** [reply]
> @krishnaTORQUE you‘ll need to take the time and create a minimal reproduction of your issue.

@hirasso sure
Here: `https://github.com/krishnaTORQUE/swup-test`

**Reply by hirasso** [reply]
Please follow the instructions found at https://docs.astro.build/en/guides/troubleshooting/#creating-minimal-reproductions
To create your reproduction on stackblitz.
